src/v0/destinations/braze/transform.js

```diff
@@ -301,7 +302,7 @@ function getPurchaseObjs(message) {
 
   const purchaseObjs = [];
 
-  if (products) {
+  if (isArray(products)) {
```

Comment:
We can `Array.isArray`(the native one) instead of lodash right ?

Reply:
If that satisfies our use-case, let's try to use native one. Also as a rule of thumb, let's native wherever it satisfies our criteria and depend on only external libraries under extreme circumstances

Reply:
Yes sure.